seatunnel-e2e/seatunnel-connector-v2-e2e/connector-cdc-postgres-e2e/src/test/resources/postgrescdc_to_postgres_with_custom_primary_key.conf

```diff
@@ -37,7 +37,7 @@ source {
     table-names = ["postgres_cdc.inventory.full_types_no_primary_key"]
     base-url = "jdbc:postgresql://postgres_cdc_e2e:5432/postgres_cdc?loggerLevel=OFF"
     decoding.plugin.name = "decoderbufs"
-    exactly_once = false
+    exactly_once = true
```

Comment:
Please update the document https://seatunnel.apache.org/docs/connector-v2/source/SqlServer-CDC

Reply:
Thanks, I forgot about this

Reply:
fixed